modules/http/src/main/scala/higherkindness/mu/http/implicits.scala

```diff
@@ -65,7 +65,10 @@ object implicits {
         decoder: Decoder[A],
         F: ApplicativeError[F, Throwable]
     ): Stream[F, A] =
-      message.body.chunks.parseJsonStream.map(_.as[A]).rethrow
+      message.body.chunks.parseJsonStream
+        .flatMap(
+          _.as[List[A]].map(Stream.emits(_)).fold(_.raiseError[Stream[F, *], A], identity(_))
```

Comment:
Need to investigate this. I know http4s changed at some point the streaming approach and started to return chunks of data instead of an array. For some reason, we're receiving here an array, instead of a single element.

Reply:
@juanpedromoreno this isn't optimal, we should be receiving a single element in the stream, not a list. I couldn't find a gap to further investigate.